values.yaml

```diff
@@ -186,3 +186,8 @@ connectInject:
     # defaults but can be customized if necessary.
     certName: tls.crt
     keyName: tls.key
+
+# Enabling rbac will create cluster roles and cluster role bindings to allow the
+# syncCatalog service to communicate with the kubernetes api to get/create services
+rbac:
+  enabled: false
```

Comment:
I think this should probably go under the `syncCatalog` section. I can see the easiest way would be:

```yaml
syncCatalog:
  rbac:
    enabled: false/true
```